requirements/base.txt

```diff
@@ -29,5 +29,6 @@ semver==2.8.1
 django-redis-cache==2.0.0
 redis==3.2.1
 html5lib==1.0.1
-zeroconf-py2compat==0.19.7
+zeroconf-py2compat==0.19.8
+ifcfg==0.21
```

Comment:
Why are we explicitly adding ifcfg back in as a dependency here? Is this not all managed by the transitive dependency in zeroconf?

Reply:
Yes, but we don't specifically pin the version number in zeroconf (to avoid conflicts for others who might be using it), so I thought given our strict version-pinning approach in Kolibri, this would be safest. Unless that's handled in another way, somehow?